components/Coverups/CoverupsDescription.jsx

```diff
@@ -0,0 +1,67 @@
+import styled from "@emotion/styled";
+import Image from "next/image";
+
+const CoverupsDescription = () => {
+  return (
+    <DescriptionContainer>
+      <div className="description-image-wrapper">
+        <Image
+          src="/Coverups/coverup-description.jpeg"
+          layout="fill"
+          objectFit="contain"
+        />
+      </div>
+      <p className="description-content">
+        Tatuaże powinni pomagać wyrażać siebie i dodawać wiary w siebie, a nie
+        na odwrót. Ale niestety, czasem tak się zdarza, że wymarzony tatuaż, z
+        różnych powodów, nie spełnia oczekiwań. Patrzenie na nieudany wzór,
+        który szpeci ciało, jest krępujące, a niekiedy wręcz żenujące. Więc
+        postanowiliśmy pomóc ludziom cieszyć się rysunkami na ich ciałach,
+        ponieważ wszyscy na to zasługują.
+      </p>
+    </DescriptionContainer>
+  );
+};
+
+const DescriptionContainer = styled.article`
```

Comment:
```suggestion
const DescriptionContainerStyled = styled.article`
```

потому что DescriptionContainer может быть и обычный компонент. Смело смотри в мои компоненты и используй мой синтакс просто чтобы не делать кашу

Reply:
Хорошо, поправил